metricbeat/include/list.go

```diff
@@ -13,10 +13,13 @@ import (
 	"github.com/elastic/beats/libbeat/logp"
 	"github.com/elastic/beats/metricbeat/helper"
 
-	// List of all metrics to make sure they are registred
+	// List of all metrics to make sure they are registered
 	// Every new metric must be added here
 	_ "github.com/elastic/beats/metricbeat/module/apache/status"
 	_ "github.com/elastic/beats/metricbeat/module/mysql/status"
+
+	// Redis module and metrics
+	_ "github.com/elastic/beats/metricbeat/module/redis"
```

Comment:
I would suggest not add add any comments and spaces between the lines here as I plan to automate this generation in the future. But for the moment ok for me.